scripts/migrations/2020-08-update_migration_created_date.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+import json
+from common import get_aws_resource, scan_table
+
+READ_ONLY_ROLE_ARN = "arn:aws:iam::975596993436:role/storage-read_only"
+dynamodb = get_aws_resource("dynamodb", role_arn=READ_ONLY_ROLE_ARN)
+s3 = get_aws_resource("s3", role_arn=READ_ONLY_ROLE_ARN)
+
+vhs_table = "vhs-storage-staging-manifests-2020-07-24"
+
+backfill_vhs_table = "vhs-storage-staging-manifests-2020-08-19"
+
+
+def get_bucket_key(item):
+    try:
+        bucket = item["payload"]["namespace"]
+        key = item["payload"]["path"]
+    except KeyError:
+        try:
+            bucket = item["payload"]["bucket"]
+            key = item["payload"]["key"]
+        except:
+            raise RuntimeError(f"Find s3 bucket, key in {item}")
+        else:
+            return bucket, key
+    else:
+        return bucket, key
```

Comment:
Two notes:

* It feels like this would be a good opportunity to tidy up the VHS structure – and use `bucket`/`key` everywhere.
* In general, bare `except:` is a bad idea in Python – it captures _everything_, including a keyboard interrupt or out-of-memory error. Normally not what you want; it's better to be more specific with the errors you catch.

Reply:
the new VHS uses bucket/key everywhere already. It's the old one that is a mixture.

I'll avoid catching everything, thanks